scss/_coverpage.scss

```diff
@@ -16,6 +16,13 @@
       }
     }
 
+    .place-name {
+      font-size: ceil($akn-font-size * 1.25); // ~ 18px
+      line-height: 1.4;
```

Comment:
h2 has `1.2`? Do we need to change h2, or this?

Reply:
```suggestion
      line-height: 1.2;
```

Reply:
Do `place-name` and `h2` need to have the same line height? 'Western Cape' and 'South Africa' are two separate things below, whereas 'Act 43 of 2011' is one thing, even if it had to run over multiple lines. At 1.2, 'Western Cape South Africa' reads as one thing.

1.2:
![image](https://user-images.githubusercontent.com/32566441/72983041-1f4c7480-3de9-11ea-8f1f-51f9071294c5.png)

1.4:
![image](https://user-images.githubusercontent.com/32566441/72983086-355a3500-3de9-11ea-995e-391df241e816.png)

Reply:
All the headings currently have the same line-height of 1.2. I think we should keep to that for now.